hack/generated/pkg/reconcilers/azure_deployment_reconciler.go

```diff
@@ -128,6 +130,8 @@ func (r *AzureDeploymentReconciler) CreateOrUpdate(ctx context.Context) (ctrl.Re
 }
 
 func (r *AzureDeploymentReconciler) Delete(ctx context.Context) (ctrl.Result, error) {
+	r.logObj("deleting resource")
```

Comment:
Ditto here - useful to log _which_ resource is being deleted.

Reply:
Same response as above -- already included